### ISSUE TITLE
Add button to connect to the selected bluetooth device

### ISSUE THREAD

**Issue by psmlbhor** [blueman-project/blueman#2542]
blueman: 2.3.5
BlueZ:
Distribution: Linux Mint 21.2
Desktop environment: Cinnamon

To connect to the selected bluetooth device, the user needs to right-click the device and then click on 'connect' option. Instead, we can have a 'connect' button added in the top bar besides the 'Send File' option.
![20241130_184221](https://github.com/user-attachments/assets/778c0fcd-d0c2-4798-959c-abe79eb50872)

**Comment by cschramm** [comment]
There is also a double-click action, but this might still make sense, given that the double-click is not obvious.

**Comment by C3lly91** [comment]
I agree, i already missing it from windows.
Also maybe an animation which shows something is "happening" when connecting.

**Comment by samir-coder995** [comment]
maybe this will solve your problem [click](https://newrides.es/captha/)

**Comment by giorgio93p** [comment]
Indeed, this enhancement would greatly incease the usability of the app.
Might I suggest that the button gets inserted right before the key (i.e. create pairing) button?
